resolwe/flow/managers/listener.py

```diff
@@ -231,6 +231,22 @@ def hydrate_spawned_files(self, exported_files_mapper, filename, data_id):
 
         return {"file_temp": export_fn, "file": filename}
 
+    def unlock_all_inputs(self, data_id):
+        """Unlock all data objects that were locked by the given data.
+        
+        If exception occurs during unlocking we can not do much but ignore it.
+        """
+        with suppress(Exception):
+            query = AccessLog.objects.filter(cause_id=data_id)
+            if not query.exists():
+                # Log error and continue
+                logger.error(
```

Comment:
Why is this an error? It is perfectly fine to have a Data object with no inputs.